uber/configspec.ini

```diff
@@ -583,6 +589,51 @@ can_volunteer   = boolean(default=True)
 consent_form    = boolean(default=False)
 
 
+[donation_tier_descriptions]
```

Comment:
``[donation_tier_descriptions]`` wasn't actually documented, so I added this as an explicit config section with some extensive comments for posterity.

One nice thing about taking Rob's suggestion about just adding a new item here is that I won't have to actually make a change to the puppet manifest; instead I can simply add a new sub-item to the ``donation_tier_items`` hash.